javascript/0071-simplify-path.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Stack
+ * https://leetcode.com/problems/simplify-path/
+ * 
+ * Time O(n) | Space O(n)
+ * @param {string} path
+ * @return {string}
+ */
+var simplifyPath = function(path) {
+    let currunt = '';
+    let myStack = [];
+    path = '/' + path + '/';
```

Comment:
Suggestion: Use interpolation

```js
path = `/${path}/`
```

Reply:
Actually. We can just remove that third if statement. if the stack is empty then nothing would happen. We don't even need to check it. I updated the code please have a look.

Reply:
yep, it's more readable. It's intimidating at first but if you try to read the code it's quite better than mine. Nicely Modularized.  I have updated the file. Please have a look.